validation-test/stdlib/FixedPointDiagnostics.swift.gyb

```diff
@@ -35,19 +35,20 @@ func test_truncatingBitPatternAPIIsStableAcrossPlatforms() {
   _ = UInt8(truncatingBitPattern: UInt(0))
   _ = UInt16(truncatingBitPattern: UInt(0))
   _ = UInt32(truncatingBitPattern: UInt(0))
-  UInt64(truncatingBitPattern: UInt(0)) // expected-error {{extraneous argument label 'truncatingBitPattern:' in call}}
+  UInt64(truncatingBitPattern: UInt(0)) // expected-error {{incorrect argument label in call (have 'truncatingBitPattern:', expected '_truncatingBits:')}}
```

Comment:
This definitely seems worse. There *is* a `UInt64.init(_: UInt)`, but now it's not picking that one. Maybe this is just "typo correction shouldn't pick names with underscores", though.

Reply:
Yeah, I was a bit surprised that these changed. It could be that something else is flaky in CSDiag and it was relying on the old behavior. Or maybe I'm wrong in thinking this is dead code, and these conversions are still reachable from the source language.

Reply:
What happens right now, with re-labeling fixes enabled, is we get 5 solutions some with labels like `exactly:`, `truncatingIfNeeded:` etc. and one without, so currently ranking ends up picking the solution without a label, so it would seem like these changes affect that?